tables/tr-g2.tbl

```diff
@@ -0,0 +1,17 @@
+#+locale:tr
+#+type:literary
+#+contraction:full
+#+grade:2
+#+dots:8
+
+# TODO: Please correct the metadata above. It is not meant to be
```

Comment:
I'll do my very best :) Would it be preferable to just change the name to tr-g2.tbl at the same time? That is, replace the current tr-g2.tbl and tr-g2.ctb with a single unified tr-g2.tbl?